packages/graphiql/src/config.ts

```diff
@@ -0,0 +1,11 @@
+export interface MedplumGraphiQLConfig {
+  introspectionUrl: string;
+}
+
+const config: MedplumGraphiQLConfig = {
+  introspectionUrl: process.env.MEDPLUM_INTROSPECTION_URL || '/schema/schema-v4.json',
```

Comment:
Prefer to leave the default value `undefined`.  That is a change in behavior, but the current default behavior is broken anyway (because it depends on `schema-v4.json` which isn't in the repo).

Reply:
That does mean that we'll need to add an env var to Github Secrets and pass it in in `.github/workflows/build.yml`